src/core/client/stream/tabs/Comments/Stream/AnsweredCommentsTab/AnsweredCommentContainer.tsx

```diff
@@ -0,0 +1,213 @@
+import { GQLUSER_STATUS } from "coral-framework/schema";
+import CLASSES from "coral-stream/classes";
+import HTMLContent from "coral-stream/common/HTMLContent";
+import Timestamp from "coral-stream/common/Timestamp";
+import { ViewConversationEvent } from "coral-stream/events";
+import {
+  SetCommentIDMutation,
+  withSetCommentIDMutation,
+} from "coral-stream/mutations";
+import { Box, Flex, Icon, TextLink } from "coral-ui/components";
+
+import { AnsweredCommentContainer_comment as CommentData } from "coral-stream/__generated__/AnsweredCommentContainer_comment.graphql";
+import { AnsweredCommentContainer_settings as SettingsData } from "coral-stream/__generated__/AnsweredCommentContainer_settings.graphql";
+import { AnsweredCommentContainer_story as StoryData } from "coral-stream/__generated__/AnsweredCommentContainer_story.graphql";
+import { AnsweredCommentContainer_viewer as ViewerData } from "coral-stream/__generated__/AnsweredCommentContainer_viewer.graphql";
+
+import { CommentContainer, UserTagsContainer } from "../../Comment";
+import ReactionButtonContainer from "../../Comment/ReactionButton";
+import { UsernameWithPopoverContainer } from "../../Comment/Username";
+
+import styles from "./AnsweredCommentContainer.css";
+
+interface Props {
+  viewer: ViewerData | null;
+  comment: CommentData;
+  story: StoryData;
+  settings: SettingsData;
+  setCommentID: SetCommentIDMutation;
+}
+
+const AnsweredCommentContainer: FunctionComponent<Props> = props => {
+  const { comment, settings, story, viewer, setCommentID } = props;
+  const banned = Boolean(
+    viewer && viewer.status.current.includes(GQLUSER_STATUS.BANNED)
+  );
+  const emitViewConversationEvent = useViewerEvent(ViewConversationEvent);
+  const onGotoConversation = useCallback(
+    (e: MouseEvent) => {
+      e.preventDefault();
+      emitViewConversationEvent({
+        from: "FEATURED_COMMENTS",
+        commentID: comment.id,
+      });
+      setCommentID({ id: comment.id });
+      return false;
+    },
+    [setCommentID, comment]
+  );
+
+  return (
+    <>
+      {comment.parent && (
+        <CommentContainer
+          viewer={props.viewer}
+          settings={props.settings}
+          comment={comment.parent}
+          story={props.story}
+          hideAnsweredTag
+          disableReplies
+          highlight
+        />
+      )}
+      <div
+        className={cn(CLASSES.featuredComment.$root, styles.root)}
+        data-testid={`featuredComment-${comment.id}`}
+      >
+        <Flex
+          direction="row"
+          alignItems="center"
+          mt={4}
+          className={CLASSES.featuredComment.authorBar.$root}
+        >
+          {comment.author && (
+            <UsernameWithPopoverContainer
+              className={CLASSES.featuredComment.authorBar.username}
+              comment={comment}
+              viewer={viewer}
+            />
+          )}
+          <Box ml={1} container="span">
```

Comment:
lot of uses of `mt` props in this component that could maybe be simplified with `HorizontalGutter` but no promises.

Reply:
I'll be honest, this was copy pasted from the original Featured comment stream. Going to prioritize the other more serious comments on the PR, but I'll see if this can be fixed. 👍